src/python/pants/backend/project_info/tasks/filedeps.py

```diff
@@ -40,21 +43,20 @@ def console_output(self, targets):
       if isinstance(concrete_target, ScalaLibrary):
         concrete_targets.update(concrete_target.java_sources)
 
-    buildroot = get_buildroot()
     files = set()
     output_globs = self.get_options().globs
 
     # Filter out any synthetic targets, which will not have a build_file attr.
     concrete_targets = set([target for target in concrete_targets if not target.is_synthetic])
     for target in concrete_targets:
-      files.add(target.address.build_file.full_path)
+      files.add(self._full_path(target.address.rel_path))
```

Comment:
This will be a directory, rather than a BUILD file name, right? Is this supposed to be conditionally using the `address.build_file` if it is set?